replica/table.cc

```diff
@@ -527,6 +527,13 @@ const std::vector<std::unique_ptr<compaction_group>>& table::compaction_groups()
     return _compaction_groups;
 }
 
+future<> table::parallel_foreach_compaction_group(std::function<future<>(compaction_group&)> action) {
+    // TODO: place a barrier here when we allow dynamic groups.
+    co_await coroutine::parallel_for_each(compaction_groups(), [&] (const compaction_group_ptr& cg) {
+       return action(*cg);
```

Comment:
Need 4 space indent, not 3.

Reply:
Note: with tablets, cg.get() may be nullptr. But let's deal with it later.

Reply:
oops, will fix indent.

Reply:
> Note: with tablets, cg.get() may be nullptr. But let's deal with it later.

you're right.